mslib/mswms/demodata.py

```diff
@@ -1029,6 +1030,30 @@ def create_server_config(self, detailed_information=False):
         (mpl_vsec_styles.VS_SpecificHumdityStyle_01, ["ecmwf_EUR_LL015"]),
         (mpl_vsec_styles.VS_TemperatureStyle_01, ["ecmwf_EUR_LL015"])
     ]
+
+
+#
+# Registration of linear layers.
+#
+# The same as above, but for 1D sections.
+register_1d_layers = None
+if mpl_1sec_styles is not None:
+    register_1d_layers = [
+        # ECMWF standard 1D section styles.
+        (mpl_1sec_styles.OS_TemperatureStyle_01, ["ecmwf_EUR_LL015"]),
```

Comment:
Mmmh.
The 1-D plot is generally simply a line. We could think about line-styles such as dashes/colors. But principally I see a different kind of configuration here...? Do we need more than one plotting class with some kind of parameter defining the standard_name of the to-be-plotted quantity?